micrometer-core/src/main/java/io/micrometer/core/instrument/binder/system/ProcessorMetrics.java

```diff
@@ -43,7 +43,7 @@ public ProcessorMetrics(Iterable<Tag> tags) {
     public void bindTo(MeterRegistry registry) {
         Runtime runtime = Runtime.getRuntime();
 
-        Gauge.builder("cpu", runtime, Runtime::availableProcessors)
+        Gauge.builder("system.cpu.count", runtime, Runtime::availableProcessors)
```

Comment:
Can we also rename this metric @mweirauch @jkschneider so it uses the hierarchy?